app/src/main/AndroidManifest.xml

```diff
@@ -74,15 +73,46 @@
 
         <activity
             android:name=".activity.BrowserActivity"
-            android:theme="@style/FullScreen">
+            android:theme="@style/FullScreen"
+            android:label="">
             <intent-filter>
                 <action android:name="org.openobservatory.ooniprobe.OPEN_BROWSER" />
-
                 <category android:name="android.intent.category.DEFAULT" />
             </intent-filter>
         </activity>
-        <activity android:name=".activity.NotificationDialog"
+        <activity
+            android:name=".activity.NotificationDialog"
             android:theme="@style/Theme.Transparent" />
+        <activity
+            android:name=".activity.RunTestActivity"
+            android:label="@string/app_name"
+            android:launchMode="singleInstance">
```

Comment:
@lorenzoPrimi do I remember it correctly that this `singleInstance` here is what saved us a lot of bugs such that the activity was recycled and so we could not open the app again for the same link? If so, maybe let's document this, because otherwise maybe in the future we don't remember why we set that flag; what do you think?

Reply:
As reference I'm reading [the link you guys circulated on slack](https://inthecheesefactory.com/blog/understand-android-activity-launchmode/en).

Reply:
Usually is a bad practice to comment the xml files in Android, I haven't seen it anywhere

Reply:
Fair enough. We should still document somewhere _why_ we start activities the way we do.

Reply:
Does XML not support comments in the form `<!-- This is a comment -->`. Why is it considered a bad practice? It's fairly common in HTML to do it.